bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/GuideEnvironment.scala

```diff
@@ -216,7 +217,9 @@ object GuideEnvironment {
   val manual: GuideEnvironment @> Option[OptsList[ManualGroup]] =
     env >=> GuideEnv.manual
 
-  private def initializeOptions(opts0: List[GuideGroup], primary0: Int): GuideEnvironment = {
+  def create(guideGroups: OptionsList[GuideGroup]): GuideEnvironment = {
+    val opts0    = guideGroups.getOptions.asScalaList
+    val primary0 = guideGroups.getPrimaryIndex.getOrElse(0).intValue
```

Comment:
Just out of curiosity, why is the `.intValue` required here for primary indices?